dot.files/bin/vol

```diff
@@ -18,6 +18,11 @@ esac
 
 VOL=${VOL/\]*}; VOL=${VOL/*\[}
 
-notify-send "Master: ${VOL}" 2> /dev/null || xmessage -nearmouse -timeout 1 "Master: ${VOL}" 2> /dev/null || ${DIALOG} --title "Vol" --infobox "Master: ${VOL}" "${G[@]}" 2> /dev/null
+if type -fP notify-send > /dev/null 2>&1; then
+	notify-send "Master: ${VOL}" 2> /dev/null\
+		|| xmessage -nearmouse -timeout 1 "Master: ${VOL}" 2> /dev/null\
+		|| ${DIALOG} --title "Vol" --infobox "Master: ${VOL}" "${G[@]}"\
+		2> /dev/null
+fi
```

Comment:
Just in-case the user doesn't have `notify-send`, they probably don't need to see the associated error. Also, this looks more presentable and therefore easier to read, I think.

Reply:
However if notify-send is not present, Xclients will receive no input on vol lvl while the intent was to default on first available method.